### PR TITLE
This week in Redox 15

### PR DESCRIPTION
**Pull request by goyox86**
Retaking the tradition of TWIR AKA This week in RedoxOS!

Only part missing is the "What’s next?" sections if  @ticki and/or @jackpot51 can give me a hint there it would be awesome!

\o/

### CODE REVIEW

File: content/news/this-week-in-redox-15.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+[TFS](github.com/redox-os/tfs) is a modular, fast, and feature rich next-gen file system, employing modern techniques for high performance, high space efficiency, and high scalability.
+
+There has been enough work done in TFS that we added this section in this edition so you can have an idea about how the progress is going there.
+
+First of all: Great news! [@ticki](https://github.com/ticki) has been working on many aspects of TFS and he considers that it is pretty near to functional. One of the great things is that as a product of this work several crates with interesting data structures and algorithms were created:
+
+- [chashmap](https://github.com/redox-os/tfs/tree/master/chashmap) - Concurrent hashmaps.
+- [cbloom](https://github.com/redox-os/tfs/tree/master/cbloom) - Concurrent bloom filters.
+[lz4](https://github.com/redox-os/tfs/tree/master/lz4) - An implementation of lz4
```
Comment (xTibor) [inline content/news/this-week-in-redox-15.md]: There's a missing bullet point.

File: content/news/this-week-in-redox-15.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+- [concurrent](https://github.com/redox-os/tfs/tree/master/concurrent) - Hazard pointers implementation.
+- TFS has a logo.
+- There is a specification document.
+- A bunch of blog posts on design: https://github.com/redox-os/tfs#resources-on-design.
+
+And according [@ticki](https://github.com/ticki) there's a ton of interesting things to write about, and eventually he will be doing an in-depth blog post about them.
+
+## Package Management
+
+There has been a ton of work on this topic specially on the (pkgutils)[https://github.com/redox-os/pkgutils]. RedoxOS packaging utilities and in the [cookbook](https://github.com/redox-os/cookbook) the collection of package recipes.
```
Comment (xTibor) [inline content/news/this-week-in-redox-15.md]: "RedoxOS" -> "Redox OS". As far as I know that's the preferred short form of the name.

**Comment by jackpot51** [comment]
Sorry I was out and did not review this - can it be merged @goyox86 ?

**Comment by goyox86** [comment]
Sure @jackpot51!

**Comment by jackpot51** [comment]
@goyox86 I recommend you post https://www.redox-os.org/news/this-week-in-redox-15/ in /r/redox and /r/rust

**Comment by goyox86** [comment]
Oh @jackpot51 I know remembered why I id ask for merge. The what's next section was not updated from the previous one. I was waiting for you for that one. Can we quickly update it. I can do a quick PR if you put here what's next. And sorry by that I did not remember!

**Comment by jackpot51** [comment]
Oh, sure. I think you can just remove that section for now.

**Comment by goyox86** [comment]
Great!

**Comment by goyox86** [comment]
Sent small PR withe the remotion of the section @jackpot51 https://github.com/redox-os/website/pull/111

**Comment by jackpot51** [comment]
@goyox86 I modified the path and title, it is now here: https://www.redox-os.org/news/this-week-in-redox-20/

**Pull request merged**